Makefile

```diff
@@ -44,10 +61,6 @@ $(foreach link,base $(JULIAHOME)/test,$(eval $(call symlink_target,$(link),$$(bu
 julia_flisp.boot.inc.phony: julia-deps
 	@$(MAKE) $(QUIET_MAKE) -C $(BUILDROOT)/src julia_flisp.boot.inc.phony
 
-# Build the HTML docs (skipped if already exists, notably in tarballs)
-$(BUILDROOT)/doc/_build/html/en/index.html: $(shell find $(BUILDROOT)/base $(BUILDROOT)/doc \( -path $(BUILDROOT)/doc/_build -o -path $(BUILDROOT)/doc/deps -o -name *_constants.jl -o -name *_h.jl -o -name version_git.jl \) -prune -o -type f -print)
-	@$(MAKE) docs
```

Comment:
Why did you delete this? If I'm not mistaken, this was necessary to avoid rebuilding docs when building and installing from tarballs. And rebuilding docs downloads Unicode.txt, which requires Internet access (disabled on distribution build machines to ensure reproducibility), and makes the build dependent on the existence of the file at the expected URL.

Reply:
This was the wrong way to write makefile dependency rules. I think it should still build it, but respecting timestamps.

Reply:
Sorry, actually this commit isn't included in 1.9.0-beta4 and yet I have a problem where docs get rebuild there, so it's caused by something different.